clang/test/dpct/usm-syclcompat.cu

```diff
@@ -0,0 +1,1090 @@
+// FIXME
+// UNSUPPORTED: system-windows
```

Comment:
Have you built the migrated result for the 2 lit cases?

Reply:
Yes.